airflow/jobs.py

```diff
@@ -186,27 +202,244 @@ def _execute(self):
         raise NotImplementedError("This method needs to be overridden")
 
 
+class DagFileProcessor(AbstractDagFileProcessor):
```

Comment:
Unfortunately, we can't move this out of the file as it depends on `SchedulerJob`.